code/src/containers/forms/product-form/components/additional-info/AdditionalInfo.tsx

```diff
@@ -56,6 +57,29 @@ const AdditionalInfo = ({
     </>
   );
 
+  const checkboxControllerRenderFunction = ({
+    field: { onChange, ...props }
+  }: {
+    field: ControllerRenderProps<ProductFormValues>;
+  }) => {
```

Comment:
Move type to separate variable for better readability

Reply:
Fixed